.travis.yml

```diff
@@ -61,23 +62,22 @@ env:
 
 matrix:
   include:
-    - php: 5.3
-      env:
-        PHPTESTSUITE='Fossology PhpUnit Test Suite'
-        CHECKBEFORE='src/vendor/bin/phpcs --standard=src/fossy-ruleset.xml src/lib/php/*/ src/www/ui/page/ src/www/ui/async/ src/spdx2 src/monk'
-        MAKETARGETS='build-lib VERSIONFILE build-cli'
     - php: 5.4
       env: PHPTESTSUITE='Fossology PhpUnit Test Suite' CHECKBEFORE='' MAKETARGETS='build-lib VERSIONFILE build-cli'
     - php: 5.5
       env: PHPTESTSUITE='Fossology PhpUnit Test Suite' CHECKBEFORE='' MAKETARGETS='build-lib VERSIONFILE build-cli'
     - php: 5.6
+      env:
+        PHPTESTSUITE='Fossology PhpUnit Test Suite' CHECKBEFORE=''
+        CHECKBEFORE='src/vendor/bin/phpcs --standard=src/fossy-ruleset.xml src/lib/php/*/ src/www/ui/page/ src/www/ui/async/ src/spdx2 src/monk'
+        MAKETARGETS='build-lib VERSIONFILE build-cli'
+    - php: 7.0
```

Comment:
I am not able to find the corresponding line on https://travis-ci.org/fossology/fossology/builds/179358115, only as allowed failure (as defined in line 79). Is this the intended behavior?

Screenshot:
![tmp](https://cloud.githubusercontent.com/assets/1187050/20703370/b26d50bc-b61b-11e6-9d29-acbada75abce.png)